content/20.treatments.md

```diff
@@ -186,7 +186,81 @@ Summarize the state of the antiviral approach.
 
 ### Broad-Spectrum Pharmaceuticals
 
-#### Hydroxychloroquine
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
+HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
+In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
+In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
+This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.
+
+###### Current Evidence
+
+Gautret et al conducted a non-randomized, non-blinded, non-placebo clinical trial on 42 hospitalized patients comparing HCQ to standard care [@doi:10.1016/j.ijantimicag.2020.105949].
+This trial found patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
+This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
+This study showed design and analysis weaknesses that severely limit interpretability of results.
+These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
+Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
+However, all other issues are fundamental design flaws and can not be corrected for.
+Thus, conclusions cannot be generalized outside of the study.
+Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.
```

Comment:
```suggestion
Because of the preliminary data presented in this study, the use of HCQ in COVID-19 treatment has subsequently been explored by other researchers.
```